### DISCUSSION TITLE
higress istio 启动失败

### DISCUSSION THREAD

**Discussion by KingOfXi** [alibaba/higress#1385] [Q&A]
参考文档https://higress.io/zh-cn/docs/ops/deploy-by-helm.html
higress部署在名为higress-system的namespace。
istio部署在名为istio-system的namespace。
下图在部署istio，其中higress-istio.yml是helm tempalte 渲染生成的。（附件已添加）]
[higress-istio .log](https://github.com/user-attachments/files/17350036/higress-istio.log)

![企业微信截图_17287355503404](https://github.com/user-attachments/assets/e43da5bc-2af6-4ed5-8f08-5afaeeeed981)

istio部署后pod未启动，日志没看出错误。

![企业微信截图_17287356757115](https://github.com/user-attachments/assets/1eab5364-844a-4827-a8df-ca9795fc95f1)

crd资源已经提前apply。
[crd-all.gen.log](https://github.com/user-attachments/files/17350045/crd-all.gen.log)
[crd-operator.log](https://github.com/user-attachments/files/17350047/crd-operator.log)

大佬能帮忙看看istio pod没启动的原因吗？

**Comment by CH3CHO** [comment]
你贴的链接已经是是过期文档了。最新的文档里已经不再提到安装 istio 的操作：https://higress.io/docs/latest/ops/deploy-by-helm/

**Reply by KingOfXi** [reply]
<img width="1019" alt="新" src="https://github.com/user-attachments/assets/87f35739-234a-4ec2-b896-d326275d7ee6">
请问下这个参数enableIstioAPI 不和higress-controller里 istio部署相关吗
